guides/headless-notification-center-guide.mdx

```diff
@@ -0,0 +1,410 @@
+---
+title: 'How to use Headless Notification Center'
+description: 'Use the headless version of the notification center'
+icon: 'circle-3'
+---
+
+## Introduction
+In this guide, you'll learn about how to use the headless version of our notification center. The headless version is useful for scenarios in which a lightweight solution for integrating notification functionality into a web app that is completely unstyled (so that you can style it as per your wish) is required.
+<br/>
+You can find the entire code ( front-end as well as back-end ) of this app [here.](https://github.com/novuhq/novu-headless-demo-app)
+
+## What is the headless notification center
+The headless version of Novu's notification library package provides users with a lightweight solution for integrating notification functionality into their web applications. With just the essential API methods, users can easily incorporate our notification system into any framework or vanilla JavaScript project, without being constrained by our default UI or dependencies.
```

Comment:
`With just the essential...`

Please start it on another paragraph..

Reply:
okay